invokeai/backend/stable_diffusion/diffusers_pipeline.py

```diff
@@ -218,15 +218,15 @@ def __call__(
 class ControlNetData:
     model: ControlNetModel = Field(default=None)
     image_tensor: torch.Tensor= Field(default=None)
-    weight: float = Field(default=1.0)
+    weight: Union[float, list[float]]= Field(default=1.0)
```

Comment:
use the type `List` rather than object type `list` here.

Reply:
Fixed